tests/unit/modules/boto_vpc_test.py

```diff
@@ -73,6 +73,21 @@ def _has_required_boto():
         return True
 
 
+def _get_moto_version():
+    '''
+    Returns the moto version
+    '''
+    try:
+        return LooseVersion(moto.__version__)
+    except AttributeError:
+        import pkg_resources
```

Comment:
No reason in particular. I just mimicked how it was in _has_required_moto(). I can pull them out, though.